src/main/java/frc/team2412/robot/RobotMap.java

```diff
@@ -1,12 +1,52 @@
 package frc.team2412.robot;
 
+import edu.wpi.first.wpilibj.ADXRS450_Gyro;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+
 //This is the class in charge of all the motors, motor ids, and any other sensors the robot uses. 
 //remember to declare robot container at the bottom of this class 
 public class RobotMap {
 
+	// Drive Motor port
+	public static final int LEFT_FRONT_ID = 0;
+	public static final int LEFT_BACK_ID = 0;
+	public static final int RIGHT_FRONT_ID = 0;
+	public static final int RIGHT_BACK_ID = 0;
+
+	// Drive Motors
+
+	private static Talon leftFront = new Talon(LEFT_FRONT_ID);
+	private static Talon leftBack = new Talon(LEFT_BACK_ID);
+	private static Talon rightFront = new Talon(RIGHT_FRONT_ID);
+	private static Talon rightBack = new Talon(RIGHT_BACK_ID);
+
+	public static SpeedControllerGroup leftSide = new SpeedControllerGroup(leftFront, leftBack);
+	public static SpeedControllerGroup rightSide = new SpeedControllerGroup(rightFront, rightBack);
```

Comment:
These should also be more descriptive.  "leftSide" could refer to many things on the robot.